manifests/concourse-manifest/concourse-base.yml

```diff
@@ -99,6 +102,7 @@ jobs:
         forward_host: 127.0.0.1
         atc:
           address: 127.0.0.1:8080
+          old_resource_grace_period: 180m
```

Comment:
I'm a bit confused now. The resources should always provide the latest version in check. That one is cached indefinitely. So why keep previous versions? Maybe it could be useful for debug (hijacking)?